core/tests/test_utils.py

```diff
@@ -3524,6 +3525,39 @@ def publish_collection(self, owner_id: str, collection_id: str) -> None:
         committer = user_services.get_user_actions_info(owner_id)
         rights_manager.publish_collection(committer, collection_id)
 
+    def link_explorations_to_story(
+        self, topic_id: str, story_id: str, *exp_ids: str
+    ) -> None:
+        """Appends a story node for each exploration id given in exp_ids to the
+        story with story_id.
+
+        Args:
+            topic_id: str. ID of the topic that contains the story.
+            story_id: str. ID of the story containing the new node.
+            *exp_ids: str. IDs of the exploration.
+        """
+        for exp_id in exp_ids:
+            story = story_fetchers.get_story_by_id(story_id)
```

Comment:
Do we need to make this db call in loop? If not, can we move it out of loop?

Reply:
@StephenYu2018 Is this thread resolved?

Reply:
@DubeySandeep I've made the necessary changes, but I've yet to push them to this PR.

Reply:
@DubeySandeep I forgot to reply back to this thread. I believe that the changes have been pushed.